front/app/hooks/useSomeFeatureFlags.ts

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from 'react';
+import {
+  currentAppConfigurationStream,
+  IAppConfiguration,
+  TAppConfigurationSetting,
+} from 'services/appConfiguration';
+import { get } from 'lodash-es';
+
+type Parameters = {
+  names: TAppConfigurationSetting[];
+  onlyCheckAllowed?: boolean;
+};
+
+// copied and modified from front/app/hooks/useFeatureFlag.ts
+export default function useSomeFeatureFlags({
```

Comment:
We should also add a default value for `onlyCheckAllowed` (`false`).

Reply:
>  I would make this useFeatureFlags, then via a prop allow for "at least one feature flag"

I'm not that sure about it. And it seems I'm not the only one 
1. https://softwareengineering.stackexchange.com/a/208228
2. https://medium.com/@amlcurran/clean-code-the-curse-of-a-boolean-parameter-c237a830b7a3

> We should also add a default value for onlyCheckAllowed (false).

Again, do you think we should change `useFeatureFlag` too?

Reply:
> I'm not that sure about it. And it seems I'm not the only one
1. https://softwareengineering.stackexchange.com/a/208228
2. https://medium.com/@amlcurran/clean-code-the-curse-of-a-boolean-parameter-c237a830b7a3

These examples look different though. 
1: you only see true or false, I agree that's not very readable. In our case, we'd have `useFeatureFlags({ onlyOneFeatureFlagRequired: true })`.
2: I didn't read the entire article, but the example at the top there looks troublesome because it's a function that's doing two things. In our case, the `onlyOneFeatureFlagRequired` check is not calling a fundamentally different functionality. It's just a parameter/config option for the same functionality. 

In any case, I didn't dive very deep into this and could be wrong. Having multiple hooks may be the better option, but this would be a new pattern in the front-end + would require a deeper look and discussion in the front-end CDM imo. If you like to discuss, feel free to add it to the CDM agenda!

> Again, do you think we should change useFeatureFlag too?
Yes, adding a default value to onlyCheckAllowed will only be a one-line change, so you can do it in useFeatureFlag too.

Reply:
(fixed)